R/randomForest_tidiers.R

```diff
@@ -1044,7 +1044,8 @@ calc_feature_imp <- function(df,
 #' TODO: not really for external use. hide it.
 #' TODO: use this other places doing similar thing.
 #' @export
-evaluate_classification <- function(actual, predicted, class, pretty.name = FALSE) {
+#' @param multi_class - TRUE when we need class and data_size, which we show for multiclass classification case.
+evaluate_classification <- function(actual, predicted, class, multi_class = TRUE, pretty.name = FALSE) {
```

Comment:
Logic looks ok. But I just noticed that the variable name 'class'. It conflicts with the base function 'class' so it should use a different name I think.

Reply:
added TODO comment.